client/web/src/components/app.tsx

```diff
@@ -78,9 +110,29 @@ function HomeView({
           data={data}
           onLoginClick={() => updateNode({ Reauthenticate: true })}
         />
+      ) : updatingViewStates.includes(updating) ? (
```

Comment:
We could also manage this with a route - if they're at the "/update" endpoint, they'll see the update view. And if there's no update available we can render something to the effect of "up to date" from that view. I know you're not looking for frontend review on this quite yet, and much of this will end up shifting with the new designs. Putting your new view at the `/update` route (if you rebase off main, you'll see our routing switch) would be a good place for it, and then Will and I can set up actually sending users to that view when desiring update (as we build out the new design).